<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GAboutPluginsDialog</class>
 <widget class="QWidget" name="GAboutPluginsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A propos des plugins</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string notr="true">Nom</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButtonConfigurer">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Configurer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_stepInfo">
         <property name="text">
          <string>Info Etape</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_redmineExport">
         <property name="text">
          <string>Export redmine</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRecharger">
       <property name="text">
        <string>Recharger</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
